tfgcv/validate_assets.go

```diff
@@ -32,11 +32,19 @@ func BuildVersion() string {
 	return buildVersion
 }
 
-// ValidateAssets instantiates GCV and audits CAI assets.
+// ValidateAssets instantiates GCV and audits CAI assets using "policies"
+// and "lib" folder under policyPath.
 func ValidateAssets(assets []google.Asset, policyPath string) (*validator.AuditResponse, error) {
+	return ValidateAssetsWithLibrary(assets,
+		filepath.Join(policyPath, "policies"),
+		filepath.Join(policyPath, "lib"))
+}
+
+// ValidateAssetsWithLibrary instantiates GCV and audits CAI assets.
+func ValidateAssetsWithLibrary(assets []google.Asset, policyPath, policyLibraryDir string) (*validator.AuditResponse, error) {
```

Comment:
Do we need to export this?

Reply:
Yes. I have a project that need to call this function directly.